modules/administration-guide/pages/configuring-storage-sizes.adoc

```diff
@@ -57,4 +57,14 @@ spec:
 <2> Specify a claim size on the next line or omit the next line to set the default claim size value. The specified claim size is only used when you select this storage strategy.
 <3> The claim size must be specified as a link:https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/[{kubernetes} resource quantity]. The available quantity units include: `Ei`, `Pi`, `Ti`, `Gi`,  `Mi` and `Ki`.
 
-pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
\ No newline at end of file
+pass:[<!-- vale RedHat.CaseSensitiveTerms = YES -->]
+
+[IMPORTANT]
+====
+Manually modifying a PVC on the cluster that was provisioned by {prod-short} is not officially supported and may have unexpected consequences.
+
+If you want to resize a PVC, you have two options:
+
+* Modify the devfile's volume sizes accordingly and restart the workspace.
+* Modify the default workspace storage size in the `CheCluster` custom resource.
```

Comment:
It might be worth linking the documentation on configuring the storage size in Che/DevSpaces. I'll find the link.

Reply:
Here's the link: https://github.com/eclipse-che/che-docs/blob/main/modules/administration-guide/pages/configuring-storage-sizes.adoc